jangaroo-tools-api/jangaroo-compiler-api/src/main/java/net/jangaroo/jooc/config/JoocOptions.java

```diff
@@ -38,7 +38,7 @@
 
   boolean isFindUnusedDependencies();
 
-  boolean isSourcesAreTests();
+  String getWarningsOutputDirectory();
```

Comment:
See above. I'd call the new option `dependencyReportOutputFile`.

Reply:
done